style.css

```diff
@@ -0,0 +1,71 @@
+body {
```

Comment:
Your stylesheet is missing a `box-sizing` declaration; see my [best practices list](https://mica-web.github.io/learn/css/best-practices#declare-box-sizing-in-all-your-stylesheets).